modules/ibm-z-disable-thp.adoc

```diff
@@ -4,71 +4,6 @@
 
 :_content-type: PROCEDURE
```

Comment:
This is no longer a procedure. I think it's closest to a concept.

It's also making the assembly that currently contains this module less useful. I think you should also add the new procedure to `scalability_and_performance/ibm-z-recommended-host-practices.adoc` or add an additional resources section to that assembly that contains a link to the Huge Pages assembly.

Reply:
s/PROCEDURE/CONCEPT/

Reply:
Added an additional resources section to that assembly that contains a link to the Huge Pages assembly.